src/Installer/GeneratedMercurialExtensions.wxs

```diff
@@ -1,71 +1,91 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-	<Fragment>
```

Comment:
It is a generated file, but not as a normal part of any build. It is rebuilt by calling a special build target, and then only when some files are known to have changed.